# Make this behave exactly the same way as mandoc(1).
# This makes my life easier for comparisions, and users won't notice.
# Changing mandoc output to "+\bo" would kill -Tps, so that's out.
$OpenBSD: patch-tmac_tty_tmac,v 1.1 2011/07/07 05:27:15 schwarze Exp $
--- tmac/tty.tmac.orig	Fri Dec 31 00:33:09 2010
+++ tmac/tty.tmac	Wed Jul  6 19:19:12 2011
@@ -17,7 +17,7 @@
 .  if c\[md] \
 .    tr \[bu]\[md]
 .
-.fchar \[bu] \z+o
+.fchar \[bu] o
 .fchar \[14] 1/4
 .fchar \[12] 1/2
 .fchar \[34] 3/4
